### ISSUE TITLE
Broken data for deep learning example on image tiles

### ISSUE THREAD

**Issue by npeschke** [scverse/spatialdata-notebooks#44]
I tried following the [deep learning example on image tiles](https://spatialdata.scverse.org/en/latest/tutorials/notebooks/notebooks/examples/densenet.html) and got a bit stuck on the data preparation step.

When I download the xenium and visium files directly as zips, the assertion is thrown that `celltype_major` is not in `xenium_sdata.table.obs` and I should try to use the linked notebook to prepare the data. However, the notebook [link](https://github.com/scverse/spatialdata-notebooks/blob/main/notebooks/paper_reproducibility/xenium_and_visium.ipynb) is broken but I found the [00_xenium_and_visium notebook](https://github.com/scverse/spatialdata-notebooks/blob/main/notebooks/paper_reproducibility/00_xenium_and_visium.ipynb) instead. That notebook also ran into some errors (see #36, #39) during execution on my machine and seemed to do a lot more than just prepare the data for the deep learning example.

Long story short, would it be possible to remove the mention of the notebook in the deep learning example and get updated xenium and visium zip files that have the `celltype_major` metadata? This would make the tutorial way more accessible.

**Comment by LucaMarconato** [comment]
Hi thanks for the feedback and for pointing out to the broken link. I will modify the notebook linking to new data to make it more accessible.

For the time being, please download the data [from here](https://drive.google.com/file/d/1G4ayVip3bEB75Hn8XJVzr3kcLy9ewEz5/view?usp=sharing). The downloaded data contains every annotation, metadata, coordinate transformation, etc except for the pure raster data. But if you copy the raster data (folders `xenium_rep1/images/morphology_mip/0`, `xenium_rep1/images/morphology_mip/1`, ..., and same for the Visium data), it should work.

Please let me know if you run into problems and I will upload the full data.

As a final note, @giovp improved the data loaders in this PR, which will be merged soon, so please notice that the APIs will slightly change and the notebook will be updated.

**Comment by LucaMarconato** [comment]
It got fixed now 😊
